index.html

```diff
@@ -3070,15 +3077,15 @@ <h4>L'hospital's Rule</h4>
                            style="width:4.375rem;display:inline;" oninput="checkfunctionsmultiple()">
                     <label>&times;</label>
                     <input type="number" id="column1" class="form__field" placeholder="n"
-                           style="width:4.375rem;display:inline;" oninput="checkfunctionsmultiple()">
+                           style="width:4.375rem;display:inline;margin-left: 5px;" oninput="checkfunctionsmultiple()">
                 </div>
                 <div style="display:inline-block" style="color:var(--apppink);">
-                    <span style="color:var(--apppink);">Matrix 2</span><br>
+                    <span style="color:var(--apppink);margin-left: 5px;">Matrix 2</span><br>
                     <input type="number" id="row2" class="form__field" placeholder="m"
-                           style="width:4.375rem;display:inline;" oninput="checkfunctionsmultiple()">
+                           style="width:4.375rem;display:inline;margin-left: 10px;" oninput="checkfunctionsmultiple()">
                     <label>&times;</label>
                     <input type="number" id="column2" class="form__field" placeholder="n"
-                           style="width:4.375rem;display:inline" oninput="checkfunctionsmultiple()">
+                           style="width:4.375rem;display:inline;margin-left: 5px;" oninput="checkfunctionsmultiple()">
```

Comment:
Use relative units such as ```vh vw rem em``` instead of ```px``` to ensure page stays responsive

Reply:
Okay Thanks for your suggestion🙂